CMakeLists.txt

```diff
@@ -215,6 +209,31 @@ if(PLASMA_TARGETS STREQUAL "Ethereal")
     add_definitions(-DSTREAM_LOGGER)
 endif(PLASMA_TARGETS STREQUAL "Ethereal")
 
+
+set_property(CACHE PLASMA_PIPELINE PROPERTY STRINGS "DirectX" "OpenGL")
```

Comment:
On OSX, it uses OpenGL3.
On Windows, it uses GLES2 emulated by DirectX11 (via ANGLE).
On Linux, it uses GLES2 (because nVidia doesn't support EGL getting a non-GLES API yet).

I would like to go with GL3 at some point, but that might be impossible until nVidia has Wayland support and full EGL support :(